08-confidence-intervals.Rmd

```diff
@@ -191,15 +215,15 @@ knitr::include_graphics("images/sampling/pennies/tactile_simulation/3_draw_at_ra
 knitr::include_graphics("images/sampling/pennies/tactile_simulation/4_put_it_back.png")
 ```
 
-**Step 5**: Repeat Steps 3 and 4 a total of 49 more times, resulting in 50 recorded years.
+**Step 5**: Repeat Steps 3 and 4 a total of `r num_pennies - 1` more times, resulting in `r num_pennies` recorded years.
```

Comment:
Love this addition of `num_pennies`! Will make it so much easier to update to something different down the road if we ever need to. Thanks!